docs/api/algorand/create-delegated-payment.mdx

```diff
@@ -0,0 +1,105 @@
+# CreateDelegatedPayment
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Request Parameters
+
+| Name | Type | Description |
+| ---- | ---- | ----------- | 
+| assetId         | `string` | The ID of the DeCash asset that is being sent. |
+| sender.address  | `string` | The address of the account that is sending the funds. Note that this account must have enough funds to cover both the amount that is being sent, and the [fee](#CreateDelegatedPayment). |
```

Comment:
`#CreateDelegatedPayment` - if you were to link some other part of a doc which is not yet created, you can either

- Create a template (empty page) in this PR
- use #TODO to find all such links later

Otherwise not sure why this fee links to the top of the page.